explore-analyze/visualize/charts/line-charts.md

```diff
@@ -0,0 +1,171 @@
+- From the **Visualize library** page by creating a new visualization.
+:::::
+
+:::::{step} Set the visualization to Line
+New visualizations default to **Bar**.
+
+Using the visualization type dropdown, select **Line**.
+:::::
+
+:::::{step} Define the data to show
+1. Select the {{data-source}} that contains your data.
+2. Drag a time field to the **Horizontal axis** and a numeric field to the **Vertical axis**. {{kib}} automatically selects an appropriate aggregation function compatible with the selected field.
+
+Optionally:
+   - Add more numeric fields to create additional series, or drag a categorical field to **Break down by** to split the series.
+   - Add a [reference line](../lens.md#add-reference-lines) to mark targets or SLOs.
+:::::
+
+:::::{step} Customize the chart to follow best practices
+You can tweak the appearance of your chart by adjusting axes, legends, and series styles from the chart settings. Consider the following best practices:
+
+**Use color wisely**
+:   Assign colors that match your users' expectations and consider your specific context.
+
+**Provide context**
+:   Add descriptive axis titles or remove them for obvious axes.
+
+For layout, hierarchy, and color guidance on dashboards, check the EUI’s [Dashboard good practices](https://eui.elastic.co/docs/dataviz/dashboard-good-practices/). 
+For more chart configuration options, go to the [Line chart settings](#settings) section.
+:::::
+
+:::::{step} Save the chart
+- If you accessed Lens from a dashboard, select **Save and return** to save the visualization and add it to that dashboard, or select **Save to library** to add the visualization to the Visualize library.
+- If you accessed Lens from the Visualize library, select **Save**. The Save menu also lets you add the visualization to a dashboard and the Visualize library.
+:::::
+
+::::::
+
+## Advanced line scenarios
+
+### Compare current versus previous period with time shift [line-previous-period]
+
+In line charts, you can enable time shift to compare the current value with a prior time range and identify deltas.
+
+![Example Lens line chart current previous](../../images/kibana-line-current-previous.png)
+
+1. Create a line chart with a time-based **Horizontal axis** and your main metric on **Vertical axis**, for example: `bytes`.
+2. From the three-dot menu in the upper-left of the layer panel, select **Duplicate layer**.  
+3. From the duplicated layer, open the **Advanced** of the **Vertical axis** and set **Time shift** to `1w`.
+   Check also [Compare differences over time](../lens.md#compare-data-with-time-offsets) for more details.
+4. Use a different color and set **Stacking** to **None** so areas overlay rather than stack.
+5. Update the legend to show “Current” and “Previous”.
+
+::::{tip}
+You can also compute the relative change using a formula, for example:  
```

Comment:
```suggestion
You can also compute the relative change by defining the axis data with a [formula](/explore-analyze/visualize/lens.md#lens-formulas), for example:  
```
We may also move this entire note inside of step 3 to avoid having to provide to much context in this note